fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/mixin/registry/sync/AccessorRegistry.java

```diff
@@ -32,4 +34,10 @@
 
 	@Accessor()
 	RegistryKey<Registry<T>> getRegistryKey();
+
+	/**
+	 * Gets the lifecycle of a registry entry.
+	 */
+	@Invoker
+	Lifecycle callMethod_31139(T object);
```

Comment:
Would prefer this having an actual name

Reply:
It is mapped in newer Yarn versions (getEntryLifecycle), but not in the one in fabric-api sadly.

Reply:
So we can keep track of this, the hash for the issue is 577ff9ccbf3ae871b351c9383064949c29e90ead

Reply:
Nvm I was literally just gonna ask you to change it as you pushed that commit lol

Reply:
Since fabric-api's mappings were updated, I was able to change this to its real name.